pkg/repository/v1_repository.go

```diff
@@ -338,3 +338,39 @@ func (r *V1Repository) fetchManifest(url string, role v1manifest.ValidManifest,
 	defer reader.Close()
 	return v1manifest.ReadManifest(reader, role, root)
 }
+
+func (r *V1Repository) ensureManifests() error {
+	// TODO
+	return nil
```

Comment:
Will this TODO will be addressed in this PR?

Reply:
supporst #230 will do this.